components/MockInterviewFormDetails.jsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { FaCircle, FaTimes } from "react-icons/fa";
+import mockInterviewFormDetails from "./MockinterviewFormData";
+
+const MockInterviewFormDetails = () => {
```

Comment:
Setup the props to read the data, the mockInterviewFormDetails should be in the parent component of this component and passes in the props for it